src/main/java/com/conveyal/gtfs/api/graphql/PatternFetcher.java

```diff
@@ -15,14 +15,51 @@
  * Created by matthewc on 3/9/16.
  */
 public class PatternFetcher {
+    public static List<WrappedGTFSEntity<Pattern>> apex(DataFetchingEnvironment env) {
+        Collection<FeedSource> feeds;
+
+        List<String> feedId = (List<String>) env.getArgument("feed_id");
+        feeds = feedId.stream().map(ApiMain::getFeedSource).collect(Collectors.toList());
+
+        List<WrappedGTFSEntity<Pattern>> patterns = new ArrayList<>();
+
+        for (FeedSource feed : feeds) {
```

Comment:
What happens if you have duplicate feed or pattern IDs in multiple feeds? The patterns are not nested under feeds.